docs/guide/blog.md

```diff
@@ -6,22 +6,17 @@ description: Learn module basics by writing and reading blog posts to your chain
 
 # Build a Blog
 
-Learn module basics by building a blockchain app to write and read blog posts.
+In this tutorial, you will learn how to create a blockchain with a module that lets you write to and read data from the blockchain. This module will implement create and read functionalities for a blog-like application. A user will be able to submit new blog posts and show a list of blog posts on the blockchain.
 
-By completing this tutorial, you will:
+The purpose of this tutorial is to guide you through the imlementation of a complete feedback loop: submitting data and reading this data back from the blockchain.
```

Comment:
love this! what is the style for abstract? we can use that style here. (and how do I look up the syntax styles for our docs? 
(I guessed at the `>` symbol... teach me!)

Reply:
```suggestion
> The purpose of this tutorial is to guide you through the imlementation of a complete feedback loop: submitting data and reading this data back from the blockchain.
```